docs/E77_EASYSOLDER.md

```diff
@@ -0,0 +1,69 @@
+# mLRS Documentation: E77 Easy Solder Boards #
+
+([back to main page](../README.md))
+
+The E77 Easy Solder Boards are designed to be an allow one with minimal soldering skills to build their own mLRS hardware. All of the parts except for the E77 module are through-hole parts to aid in assembly.  These boards are flexible in that they can be used as a Tx module, Rx module or a SiK replacement. Additionally, these boards support diversity when paired with an additional E22 module.
+
+<table>
+  <tbody>
+    <tr>
+      <td>Frequency Bands</td>
+      <td>868 MHz/915 MHz (E77-900M22S) or 433 MHz/70 cm (E77-400M22S)</td>
+    </tr>
+    <tr>
+      <td>Max. RF Output Power</td>
+      <td>22 dBm (158 mW)</td>
+    </tr>
+    <tr>
+      <td>Supported Modes</td>
+      <td>31 Hz, 19 Hz</td>
+    </tr>
+  </tbody>
+</table>
+
+### As Tx Module ###
+
+Connections (name in respect to board print-ons, otherwise please refer to graphic):
+
+- serial: Tx, Rx
+- com/cli: Tx, Rx
+- debug: STx
```

Comment:
I guess I would list it last, at least after S

Reply:
Debug last